sql/migrations/20200518160736_world.sql

```diff
@@ -0,0 +1,60 @@
+delimiter ??
+CREATE PROCEDURE `add_migration`()
+BEGIN
+DECLARE v INT DEFAULT 1;
+SET v = (SELECT COUNT(*) FROM `migrations` WHERE `id`='20200518160736');
+IF v=0 THEN
+INSERT INTO `migrations` VALUES ('20200518160736');
+-- Add your query below.
+
+
+-- add cleaver to cook ghilm
+UPDATE `creature_equip_template` SET `equipentry1` = 2827 WHERE `entry` = 1355;
+
+-- pathing and scripts for cook ghilm
+UPDATE `creature` SET `movement_type`= 2 WHERE `id`= 1355;
+INSERT INTO `creature_movement_template` (`entry`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `waittime`, `wander_distance`, `script_id`) VALUES
+(1355, 1, -5666.22, -1565.44, 383.204, 2.32129, 10000, 0, 0),
+(1355, 2, -5666.22, -1565.44, 383.204, 2.32129, 500, 0, 1356),
+(1355, 3, -5664.5615, -1566.6590, 383.2048, 5.9077, 7000, 0, 1355);
+INSERT INTO `creature_movement_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES (1355, 3, 1, 25, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'cook ghilm - point');
+INSERT INTO `creature_movement_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES (1355, 4, 19, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'cook ghilm - set default equip');
+INSERT INTO `creature_movement_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES (1356, 0, 19, 0, 0, 0, 0, 0, 0, 0, 0, 2196, 0, 0, 0, 0, 0, 0, 0, 0, 'cook ghilm - equip meat leg');
+
+-- pathing for frast dokner
+UPDATE `creature` SET `movement_type`= 2 WHERE `id`= 1698;
+INSERT INTO `creature_movement_template` (`entry`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `waittime`, `wander_distance`, `script_id`) VALUES
+(1698, 1, -5714.37, -1589.52, 383.226, 1.55334, 40000, 0, 0),
+(1698, 2, -5711.45, -1590.56, 383.204, 100, 0, 0, 0),
+(1698, 3, -5712.14, -1596.03, 383.204, 0.461232, 60000, 0, 0);
+
+-- pathing and scripts for kazan mogosh
+UPDATE `creature` SET `movement_type`= 2 WHERE `id`= 1237;
+INSERT INTO `creature_movement_template` (`entry`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `waittime`, `wander_distance`, `script_id`) VALUES
+(1237, 1, -5671.19, -1577.15, 383.329, 3.4383, 40000, 0, 0),
+(1237, 2, -5668.92, -1574.36, 383.204, 100, 0, 0, 0),
+(1237, 3, -5668.73, -1571.90, 383.204, 100, 0, 0, 0),
+(1237, 4, -5665.08, -1567.92, 383.204, 0.753848, 50000, 0, 0);
+(1237, 5, -5665.08, -1567.92, 383.204, 0.753848, 0, 0, 1237);

```

Comment:
Here you can use a single cmove script ID: 11 (generic run) to return to the starting point in a runing state.

Reply:
Thanks, that worked perfectly.